.github/workflows/cli.yml

```diff
@@ -70,6 +70,7 @@ jobs:
           npx prettier --check .
 
   lint:
+    name: Lint and run unit tests
```

Comment:
The reasoning for grouping the two is that we'd be compiling again with a separate job for unit tests, but the downside is that it blocks the  other jobs that depend on lint until tests are done. Happy to change if that sounds better.